Functions/QualityControl/xASL_qc_CollectSoftwareVersions.m

```diff
@@ -80,6 +80,12 @@
         Software.ExploreASL = [Fname(9:end) Fext];
     end
     
+
+    %% Get ExploreASL commit for optimal provenance
+    [~, xASL_gitCommit] = xASL_system(['cd ' x.opts.MyPath '; git rev-parse HEAD']);
```

Comment:
What if you run that on Windows, will it work?

What if you don't have git installed? Then it will return "command not found" - so that error message has to be parsed.

Reply:
Good point! Will check this